static/src/javascripts/lib/cookies.js

```diff
@@ -1,4 +1,9 @@
 // @flow
+const ERR_INVALID_COOKIE_NAME = `Cookie must not contain invalid characters (space, tab and the following characters: '()<>@,;"/[]?={}')`;
+
+const isValidCookieValue = (name: string): boolean =>
+    !/[()<>@,;"\\/[\]?={} \t]/g.test(name);
```

Comment:
might be handy to add a comment explaining what the regex is checking for?

Reply:
Agree